sigopt/objects.py

```diff
@@ -137,13 +136,13 @@ def dumps(obj, indent_level=0):
     indent = ' ' * indent_level
 
     if isinstance(obj, BaseApiObject):
-      return six.u('{0}{1}'.format(indent, str(obj).replace('\n', '\n{0}'.format(indent))))
+      return '{0}{1}'.format(indent, str(obj).replace('\n', '\n{0}'.format(indent)))
     if is_mapping(obj):
       if obj:
-        return six.u('{0}{\n{1},\n{0}}').format(
+        return '{0}{{\n{1},\n{0}}}'.format(
```

Comment:
this one was a little clunky, it required escaping the literal `{` with another `{`

Reply:
FWIW we can consider FSLs in the (near) future to clean this up even further, but wanted to isolate this change to just `six` removal

Reply:
I ran into this in https://github.com/sigopt/sigopt-python/pull/191 and I determined that the original was just broken

Reply:
lol, completely forget we already checked that. and that was less than 2 months ago. whoops.